<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>GEOsearcher</class>
 <widget class="QMainWindow" name="GEOsearcher">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1373</width>
    <height>751</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>GEOsearcher</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPushButton" name="search_button">
    <property name="geometry">
     <rect>
      <x>1180</x>
      <y>40</y>
      <width>93</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>search</string>
    </property>
   </widget>
   <widget class="QComboBox" name="comboBox">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>40</y>
      <width>87</width>
      <height>22</height>
     </rect>
    </property>
    <item>
     <property name="text">
      <string>Any</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>miRNA</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>lncRNA</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>mRNA</string>
     </property>
    </item>
   </widget>
   <widget class="QLineEdit" name="terms">
    <property name="geometry">
     <rect>
      <x>430</x>
      <y>40</y>
      <width>741</width>
      <height>21</height>
     </rect>
    </property>
   </widget>
   <widget class="QScrollArea" name="scrollArea">
    <property name="geometry">
     <rect>
      <x>89</x>
      <y>86</y>
      <width>1191</width>
      <height>571</height>
     </rect>
    </property>
    <property name="widgetResizable">
     <bool>true</bool>
    </property>
    <widget class="QWidget" name="scrollAreaWidgetContents">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>1189</width>
       <height>569</height>
      </rect>
     </property>
     <widget class="QTableWidget" name="tableWidget">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>1191</width>
        <height>571</height>
       </rect>
      </property>
     </widget>
    </widget>
   </widget>
   <widget class="QComboBox" name="comboBox_2">
    <property name="geometry">
     <rect>
      <x>190</x>
      <y>40</y>
      <width>231</width>
      <height>22</height>
     </rect>
    </property>
    <item>
     <property name="text">
      <string>Any</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Homo sapiens</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Mus musculus</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Arabidopsis thaliana</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Rattus norvegicus</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Drosophila melanogaster</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Saccharomyces cerevisiae</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Caenorhabditis elegans</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Escherichia coli</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Danio rerio</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Oryza sativa</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>synthetic construct</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Bos taurus</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Schizosaccharomyces pombe</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Sus scrofa</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Gallus gallus</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Salmonella enterica</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Staphylococcus aureus</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Zea mays</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Macaca mulatta</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Pseudomonas aeruginosa</string>
     </property>
    </item>
   </widget>
   <widget class="QLineEdit" name="lineEdit">
    <property name="geometry">
     <rect>
      <x>630</x>
      <y>670</y>
      <width>31</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>1</string>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>670</x>
      <y>670</y>
      <width>21</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>of</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>700</x>
      <y>670</y>
      <width>41</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>1</string>
    </property>
   </widget>
   <widget class="QPushButton" name="Prev_button">
    <property name="geometry">
     <rect>
      <x>520</x>
      <y>670</y>
      <width>93</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Prev</string>
    </property>
   </widget>
   <widget class="QPushButton" name="Next_button">
    <property name="geometry">
     <rect>
      <x>730</x>
      <y>670</y>
      <width>93</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Next</string>
    </property>
   </widget>
   <widget class="QPushButton" name="Last_button">
    <property name="geometry">
     <rect>
      <x>830</x>
      <y>670</y>
      <width>93</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Last</string>
    </property>
   </widget>
   <widget class="QPushButton" name="First_button">
    <property name="geometry">
     <rect>
      <x>420</x>
      <y>670</y>
      <width>93</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>First</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="checkBox">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>670</y>
      <width>111</width>
      <height>19</height>
     </rect>
    </property>
    <property name="text">
     <string>select all</string>
    </property>
   </widget>
   <widget class="QPushButton" name="download_button">
    <property name="geometry">
     <rect>
      <x>1190</x>
      <y>670</y>
      <width>93</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Download</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1373</width>
     <height>26</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
